docs/guides/hedgehog-engine/rangers/models/import.mdh

```diff
@@ -0,0 +1,54 @@
+---
+description: Model Importing guide for Sonic Frontiers
+---
+# Model Importing
+
+!!! info
+    This guide uses the following tools:
+
+    - [HedgeArcPack](/tools/hedgehog-engine/common/files/#hedgearcpack){ target="_blank"}
+    - [Blender](https://www.blender.org/download/)
+    - [ModelConverter.exe](https://hedgedocs.com/tools/general/files/#skythtools)

```

Comment:
Skyth's new ModelConverter is now listed on the Hedgehog Engine Model tools page. Please reference that instead.